srcs/runners_manager/vm_creation/aws/AwsManager.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         )
         self.tags = settings.get("tags")
         self.ec2 = boto3.client("ec2")
+        logger.debug('AWS runner-manager init')
```

Comment:
Take the opportunity to add a bit more detail, like, `AWS init, configured with region: x and tags: x,y,z`